source/transfer_list.rst

```diff
@@ -820,6 +821,48 @@ and reserved for use as a TPM Command Response Buffer interface.
      - Size of CRB.
 
 
+.. _rw_mem_layout:
+
+Read-Write Memory Layout Entry Layout (XFERLIST_RW_MEM_LAYOUT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This type defines the layout of a region of read-write memory that a client can
+allocate into its memory map. If other memory types are required (i.e.
+read-only memory), separate TE's should be defined.
+
+.. _tab_sram_mem_conf:
+.. list-table:: Layout for a RW memory layout entry
+   :widths: 2 5 5 6
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **7**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - The size of the layout in bytes.
+
+   * - addr
+     - `sizeof(uintptr_t)`
```

Comment:
The TL is architecture-independent so you should avoid references to machine register sizes and use exact widths wherever possible.